lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -1,4 +1,5 @@
 export fluxnet_simulation
+include(joinpath(ClimaLandSimulations_dir, "src", "utilities", "pfts.jl"))

```

Comment:
I think you should include in the module instead lib/ClimaLandSimulations/src/ClimaLandSimulations.jl